test/tsickle_test.ts

```diff
@@ -104,6 +104,32 @@ describe('emitWithTsickle', () => {
     expect(jsSources['b.js']).toContain(`export { Foo } from './a';`);
   });
 
+  it('should not add comments when using reolveJsonModule in json files', () => {
+    const tsSources = {
+      'a.ts': `
+        import x from './b.json';
+        export { x };
+      `,
+      'b.json': '{"foo": "bar"}',
+    };
+
+    const tmp = path.join(process.cwd(), '.tmp');
```

Comment:
Why is this needed?  The other tests don't need a temp directory.

Reply:
Cause otherwise there will be an error
that I am trying to override the source file. Since after transformation json remains json, while the Ts files will become js files thus they don’t emit this error.

Reply:
Can you clarify what the error is?  (Depending on the environment where the tsickle tests run, I think this process.cwd() won't work, so we should figure out some other way of making this test work.)

Reply:
The error is `TS5055: Cannot write file .../test/unit_test.runfiles/tsickle/b.json' because it would overwrite input file.`

Reply:
I pushed a fixed. The problem was that no outDir is specified it will emit the transpiled code in the same folder of the source.

That is not a problem when having `ts` to `js` as when transpiling this will not override the source file as it will generate another file. However when having `json` files in the pipeline it will mean that the emitted file will override the source file as they are both named the same with the same extention.